tests/dialects/test_snowflake.py

```diff
@@ -648,40 +652,40 @@ def test_table_literal(self):
         # All examples from https://docs.snowflake.com/en/sql-reference/literals-table.html
         self.validate_all(
             r"""SELECT * FROM TABLE('MYTABLE')""",
-            write={"snowflake": r"""SELECT * FROM TABLE('MYTABLE')"""},
+            write={"snowflake": r"""SELECT * FROM IDENTIFIER('MYTABLE')"""},
```

Comment:
These two forms are equivalent -- the tests were changed in because `TABLE(...)` is now parsed into an `exp.ToIdentifier` expression and its first `_sql_name` is `IDENTIFIER`, instead of table.